src/model/ai/AlphaBetaSearchNode.java

```diff
@@ -38,22 +42,18 @@ public AlphaBetaSearchNode(Board board, int depthLevel, PieceColor currentPlayer
         return this.children;
     }
 
-    public Board getBoard() {
-        return this.board;
-    }
-
     public Move getMoveThatGotToThisState() {
         return this.moveThatGotToThisState;
     }
 
-    public boolean isLeaf() {
-        return (this.depthLevel == 0 || this.board.isEndState());
-    }
-
     public double getValue() {
         return this.value;
     }
 
+    public boolean isLeaf() {
+        return (this.depthLevel == 0 || this.board.isEndState());
+    }
+
     public void setValue(double value) {
         this.value = value;
     }
```

Comment:
Yeah, I have it set to reorganize imports, as well as many additional actions in the Java > Editor > Save Actions window under Preferences. Here are my additional actions:

```
Add 'this' qualifier to unqualified field accesses
Add 'this' qualifier to unqualified method accesses
Convert control statement bodies to block
Remove unused imports
Add missing '@Override' annotations
Add missing '@Override' annotations to implementations of interface methods
Add missing '@Deprecated' annotations
Remove unnecessary casts
Sort all members
Remove trailing white spaces on all lines
Correct indentation
```

I think the one of note is the sort all members.